TECHNICAL_DESIGN.md

```diff
@@ -1,21 +1,18 @@
 # Technical Design Documentation
 
-This document provides detailed technical information about the HeyMax SubCaps Viewer implementation, covering both the Chrome Extension and Tampermonkey userscript versions.
+This document provides detailed technical information about the HeyMax SubCaps Viewer Tampermonkey userscript implementation.
 
 ## Architecture Overview
 
-The HeyMax SubCaps Viewer uses network request interception to capture transaction data from the HeyMax API, store it locally, and calculate SubCap spending for UOB credit cards. Two implementation variants exist:
+The HeyMax SubCaps Viewer uses network request interception to capture transaction data from the HeyMax API, store it locally, and calculate SubCap spending for UOB credit cards.
 
-1. **Chrome Extension** (Manifest V3) - Chrome/Edge browsers
-2. **Tampermonkey Userscript** - Cross-browser compatible
-
-Both implementations share the same core functionality but differ in deployment mechanism and storage APIs.
+The Tampermonkey userscript is cross-browser compatible and works on desktop browsers (Chrome, Firefox, Safari, Opera, Edge) as well as mobile browsers that support Tampermonkey (Edge Mobile recommended, also Firefox Mobile and Kiwi Browser on Android).
```

Comment:
same as the previous comment, HeyMax.ai site doesn't load on Firefox mobile and kiwi browser